llama_stack/strong_typing/schema.py

```diff
@@ -0,0 +1,752 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+"""
+Type-safe data interchange for Python data classes.
+
+:see: https://github.com/hunyadi/strong_typing
+"""
+
+import dataclasses
+import datetime
+import decimal
+import enum
+import functools
+import inspect
+import json
+import typing
+import uuid
+from copy import deepcopy
+from typing import (
+    Any,
+    Callable,
+    ClassVar,
+    Dict,
+    List,
+    Literal,
+    Optional,
+    Tuple,
+    Type,
+    TypeVar,
+    Union,
+    overload,
+)
+
+import jsonschema
```

Comment:
I believe this also needs a corresponding `uv add jsonschema` to update pyproject.toml / uv.lock, and probably a manual update of requirements.txt however that's typically kept in sync with the other files.

Otherwise, I get this error when attempting to execute a `llama stack build` locally using this PR's changes in a fresh virtual env:
```
  File "/home/bbrownin/tmp/llama-stack/llama_stack/strong_typing/schema.py", line 38, in <module>
    import jsonschema
ModuleNotFoundError: No module named 'jsonschema'
```

Reply:
@bbrowning ooh, _great_ catch!